src/components/gds-media-card/gds-media-card.tsx

```diff
@@ -38,20 +38,19 @@ export class GdsMediaCard {
   @Prop() description: string
 
   render() {
-    console.log('img', this.imageUrl)
-    console.log('headline', this.headline)
     // Main content
     const card = (
-      <div class="card">
+      <gds-card>
         {/* TODO: Implement a gds-image component that implements <picture> and srcset etc. */}
         <img src={this.imageUrl} class="image" />
         <div class="headline">
           <gds-heading size="s">{this.headline}</gds-heading>
         </div>
+        <gds-paragraph class="description">{this.description}</gds-paragraph>
```

Comment:
I feel like a wrapper with padding would be better than applying styles to the paragraph. For example perhaps this would eventually support multiple paragraphs.

Reply:
I don't think wrapper with padding would remove the p element's standard margins? I think the best option would be to add `gds-text` or `gds-body` which would basically be the same as `gds-paragraph` but without margins.

Reply:
A common solution for card content I think would be using the wrapper:

```
.card-content-wrapper > *:first-child { margin-top: 0; }
.card-content-wrpaper > *:last-child { margin-bottom: 0; }
```
or

```
.card-content-wrapper > p { margin: 0; }
```

`gds-text` sounds good though.

Reply:
@henritukiainen the tricky part is that now that all the components are using shadow dom, then AFAIK I can't target the p element from outside the gds-paragraph component. So I'm suggesting to use gds-text at some point.